.github/component_owners.yml

```diff
@@ -32,6 +32,7 @@ components:
     - dehaansa
     - Mrod1598
     - rmfitzpatrick
+    - breedx-splk
```

Comment:
Can you add yourself to the [README](https://github.com/open-telemetry/opentelemetry-java-contrib/tree/main/jmx-metrics#component-owners) as well?

Reply:
Actually, wait, where exactly in the readme?